src/Compilers/CSharp/Portable/Binder/WithCrefTypeParametersBinder.cs

```diff
@@ -63,6 +63,7 @@ private MultiDictionary<string, TypeParameterSymbol> CreateTypeParameterMap()
                 case SyntaxKind.IndexerMemberCref:
                 case SyntaxKind.OperatorMemberCref:
                 case SyntaxKind.ConversionOperatorMemberCref:
+                case SyntaxKind.ExtensionMemberCref:
```

Comment:
>case SyntaxKind.ExtensionMemberCref: [](http://example.com/codeflow?start=16&length=36)

Is this change related to "Addresses part of [https://github.com/dotnet/roslyn/issues/78967](https://github.com/dotnet/roslyn/issues/78967) ("We want unqualified CREF to work within the context of enclosing static type cref=`"extension(int).M"`")"? If not, consider adjusting PR description accordingly.